src/_P106_BME680.ino

```diff
@@ -64,29 +64,18 @@ boolean Plugin_106(uint8_t function, struct EventStruct *event, String& string)
       break;
     }
 
+    case PLUGIN_I2C_HAS_ADDRESS:
     case PLUGIN_WEBFORM_SHOW_I2C_PARAMS:
     {
-      uint8_t choice = PCONFIG(0);
-
-      /*
-         String options[2];
-         options[0] = F("0x76 - default settings (SDO Low)");
-         options[1] = F("0x77 - alternate settings (SDO HIGH)");
-       */
-      int optionValues[2] = { 0x77, 0x76 };
-      addFormSelectorI2C(F("i2c_addr"), 2, optionValues, choice);
-      addFormNote(F("SDO Low=0x76, High=0x77"));
-      break;
-    }
-
-    #if USE_I2C_DEVICE_SCAN
-    case PLUGIN_I2C_GET_ADDRESSES_HEX:
-    {
-      string = F("67,77"); // List of device addresses, hex, comma separated, _no_ 0x prefix
-      success = true;
+      const int i2cAddressValues[] = { 0x77, 0x76 };
```

Comment:
Is there a reason to have these addresses swapped?
e.g. is 0x77 the default on many boards?

Reply:
The string had a typo, and the original has `0x77, 0x76`, so 0x77 seems to be the default.